docs/core/tools/dotnet.md

```diff
@@ -240,7 +240,7 @@ Starting with .NET Core SDK 2.1.300, a number of tools that were available only
 | dev-certs                                         | Creates and manages development certificates.                |
 | [ef](/ef/core/miscellaneous/cli/dotnet)           | Entity Framework Core command-line tools.                    |
 | [user-secrets](/aspnet/core/security/app-secrets) | Manages development user secrets.                            |
-| [watch](/aspnet/core/tutorials/dotnet-watch)      | Starts a file watcher that runs a command when files change. |
+| [watch](dotnet-watch.md)                          | Starts a file watcher that runs a command when files change. |
```

Comment:
Is the description still accurate?

Reply:
Good question. The command's function hasn't changed, but the description was minimal, so I expanded it.